### PR TITLE
feat(breadcrumb): implement component

### PR DESCRIPTION
**Pull request by dpellier**
- update ods-link to allow icon-only link (waiting for design input on this as the rendering on breadcrumb is strange)
- update doc generator to hide non public Prop (and update some component accordingly)
- update doc generator to handle multiple sub-components
- KNOWN ISSUE: unable to prevent breadcrumb link on documentation page to not redirect on click (won't fix here, should be worked on during Doc improvement task)

### CODE REVIEW

**Review by feoche — CHANGES_REQUESTED** [review]
### Documentation
- [x] Missing differentiating between Breadcrumb and BreadcrumbItem props
- [x] Missing differentiating between Breadcrumb and BreadcrumbItem events

### Style
- [ ] Pipe character "|" should have : `font-size: 1.1rem; line-height: 1.5rem; font-weight: 700`
- [ ] Add `aria-current="page"` attribute to last BreadcrumbItem (see https://a11y-style-guide.com/style-guide/section-navigation.html#kssref-navigation-breadcrumbs)

**Review by feoche — CHANGES_REQUESTED** [review]
- [x] Vertical alignment issue, font-sizes are fine (see example, top is staging, bottom is expected)
image

**Review by aesteves60 — CHANGES_REQUESTED** [review]

File: packages/ods/src/components/breadcrumb/src/components/ods-breadcrumb/ods-breadcrumb.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
+$breadcrumb-spacing: 8px;
+
+:host(.ods-breadcrumb) {
+  display: flex;
+  flex-wrap: wrap;
+  column-gap: $breadcrumb-spacing;
+  align-items: center;
```
Comment (feoche) [inline packages/ods/src/components/breadcrumb/src/components/ods-breadcrumb/ods-breadcrumb.scss:7]:
```suggestion
  align-items: baseline;
  line-height: 1rem;
```
does fix the global alignment with icons:
Before : 
image

After :
image

**Review by feoche — CHANGES_REQUESTED** [review]

**Review by aesteves60 — APPROVED** [review]

File: packages/ods/src/components/breadcrumb/src/components/ods-breadcrumb-item/ods-breadcrumb-item.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+  @Prop({ reflect: true }) public href!: string;
+  @Prop({ reflect: true }) public icon?: OdsIconName;
+  @Prop({ reflect: true }) public isDisabled: boolean = false;
+  @Prop({ reflect: true }) public label?: string;
+  @Prop({ reflect: true }) public referrerpolicy?: ReferrerPolicy;
+  @Prop({ reflect: true }) public rel?: HTMLAnchorElement['rel'];
+  @Prop({ reflect: true }) public target?: HTMLAnchorElement['target'];
+
+  @Event() odsBreadcrumbItemClick!: EventEmitter<globalThis.Event>;
```
[the remark this answers is not in the document]
Comment (dpellier) [reply packages/ods/src/components/breadcrumb/src/components/ods-breadcrumb-item/ods-breadcrumb-item.tsx]: Event is already imported as Stencil decorator, updated all to `MouseEvent` anyway

**Review by feoche — APPROVED** [review]

**Review by Leotheluck — APPROVED** [review]

**Pull request merged**